app/controllers/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
```

Comment:
You'll want to make sure a signed in user cannot access this path as well. You can try `redirect_to root_path and return if current_user` before `@user = User.new`